life_expectancy/cleaning.py

```diff
@@ -2,46 +2,85 @@
 Script to load, clean and save data
 """
 import argparse
+import pathlib
 import pandas as pd
 
 
 
-# life_path = pathlib.Path.cwd()
-# data_path = life_path.joinpath('data')
+PARENT_PATH = pathlib.Path(__file__).parent
+FILE_PATH = PARENT_PATH / 'data'
+INPUT_FILE_NAME = 'eu_life_expectancy_raw.tsv'
+OUTPUT_FILE_NAME = 'pt_life_expectancy.csv'
+INPUT_FILE_PATH = FILE_PATH / INPUT_FILE_NAME
+OUTPUT_FILE_PATH = FILE_PATH / OUTPUT_FILE_NAME
 
-def clean_data(path: str, country: str = 'PT'):
+
+def load_data(file_path: str)-> pd.DataFrame:
     """
-    Function to clean data
+        Function to load data from data folder, given a file_name
     """
-
-    df_expectancy = pd.read_csv(path, sep = '\t') # data_path.joinpath(file_name)
-
+    try:
+        df_expectancy = pd.read_csv(file_path, sep = '\t')
+    except FileNotFoundError: #pragma: no cover
+        print('File not found!')
+    except pd.errors.ParserError: #pragma: no cover
+        print('Error parsing the file!')
```

Comment:
Careful with this: remember that raising an exception and not doing anything with it is a code smell